### DISCUSSION TITLE
Azuracast station running but no audio recieved

### DISCUSSION THREAD

**Discussion by rogercreagh** [AzuraCast/AzuraCast#6912] [Q&A]
Continuing my nightmare of trying to get a very basic azuracast install working, I finally got would looked like a working system but although the station appeared to be playing - a song was listed on the public page screen with the time line advancing - no audio was being received on an device that accessed it.

I wondered if this might be a firewall problem so reset it to enable all the 8000 ports for TCP again. Since that made no change I tried restarting the server - which of course re-enabled the default apache and caused all sorts of horrible attacker lockout messages in edge browser when trying to access the site. So I stopped Apache service again and now it is simply refusing to connect as if ngnix is not running.

Is there a list of docker.sh commands somewhere?

 how do I use it to check the status and find out what is going on?

Trying to clear the cache with  `sudo ./docker.sh cli cache:clear`
gives the message `service "web" is not running` - does this indicate that docker has stopped working? How do I start it again?

**Comment by gAlleb** [comment]
To run Azuracast after reboot we run
```
cd /var/azuracast
sudo docker compose up -d
```
If you don't hear sound yyou could try and
1. Open Icecast ports.
2. You could also try and switch on "System Administration" - "System Settings" - "Use Web Proxy for Radio" in web interface.
